app/src/main/java/com/vero/tasky/auth/presentation/login/LoginScreen.kt

```diff
@@ -0,0 +1,62 @@
+package com.vero.tasky.auth.presentation.login
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.vero.tasky.R
+import com.vero.tasky.auth.presentation.components.BaseAuthScreen
+import com.vero.tasky.auth.presentation.components.PasswordTextField
+import com.vero.tasky.core.presentation.components.LoadingTextButton
+import com.vero.tasky.core.presentation.components.VerifiableTextField
+import com.vero.tasky.ui.theme.bottomText
+import com.vero.tasky.ui.theme.bottomTextAccent
+
+@Composable
+fun LoginScreen(
+    navigateTo: (route: String) -> Unit,
```

Comment:
I personally don't let my screens know to which screen they are going to move next

I prefer having lambdas here

Ex:

onCreateAccount: () -> Unit (Triggered once the user clicks on "Dont have an account"
onLogIn() -> Unit (Triggered once the login was successful)


And then, have the Navigation (MainActivity, or wherever it's hosted) be in charge of deciding what screen to show next